Add instructions on how to remove a backup

ssemyan opened the pull request:
Would be good to add instructions on how to remove a backup since you cannot do it via the portal, and if a recovery vault has a backup in it, it can't be deleted. Thus if you try out Azure Backup, it is hard to undo (and thus you will be charged for it).

cjgronlund commented:
@benjguin 
Thank you for your detailed contribution this this article. The information has been incorporated by the writer as appropriate. A new HBase SDK for .NET has recently been released: https://github.com/hdinsight/hbase-sdk-for-net
Thanks again,

Carolyn
Azure Documentation

Blackmist commented:
The Storm changes look good, and have been merged in. Should appear in this repo on next publish.

ssemyan committed:
Add instructions on how to remove a backup

Would be good to add instructions on how to remove a backup since you cannot do it via the portal, and if a recovery vault has a backup in it, it can't be deleted. Thus if you try out Azure Backup, it is hard to undo (and thus you will be charged for it).
